commands/build/lib/config.js

```diff
@@ -53,6 +53,20 @@ const getExternalDefault = ({ pkg }) => {
   return Object.keys(peers);
 };
 
+const getExternalCore = ({ pkg }) => {
+  const defaultExternal = [
+    '@nebula.js/stardust',
+    'picasso.js',
+    'picasso-plugin-q',
+    'react',
+    'react-dom',
+    /^(?!@qlik-trial\/qmfe-data-client-parcels)(@qlik-trial\/qmfe-)/,
+    /^@qlik\/api\//,
+  ];
+  const peers = Object.keys(pkg.peerDependencies || {});
+  return [...defaultExternal, ...peers];
+};
+
```

Comment:
This is needed (the regex) to match for example "@qlik/api/reloads", and I don't think adding the others is an issue.

Reply:
you have verified that adding "@qlik/api" as peerDependencie in core package.json did not exclude it?

Reply:
yea I did, but after our discussion I will get rid of this anyways.